lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -324,6 +324,22 @@ def max_thread_limit
   Rake::Task[:spec_clean].invoke
 end
 
+begin
+  require 'parallel_tests'
+
+  args = ['-t', 'rspec']
+  Dir.glob(pattern).each { |f| args << f }
+
+  desc "Parallel spec tests"
+  task :parallel_spec do
+    Rake::Task[:spec_prep].invoke
+    ParallelTests::CLI.new.run(args)
+    Rake::Task[:spec_clean].invoke
+  end
+rescue LoadError
+  # ignore
```

Comment:
I think the empty rescues are intentional, otherwise I might as well make `parallel_tests` a dependency in the gemspec.  I've done it this way so that people aren't forced to download that Gem if they don't want to use it.